server/tests/stt_tt_parse_content_api_test.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import unittest
+
+from stt.io.feed_parsers.stt_tt_parse_content_api import ContentAPITTItemParser
+
+
+def fixture(filename):
+    return os.path.join(os.path.dirname(__file__), "fixtures", filename)
+
+
+class ContentAPITTItemParserTestCase(unittest.TestCase):
+    def setUp(self):
+        self.parser = ContentAPITTItemParser()
+
+        # Load test fixture
+        with open(fixture("api/stt_tt_content_api.json")) as _file:
+            self.fixture_data = json.load(_file)
+
+    def test_parser_with_fixture_data(self):
+        """Test parser with real fixture data."""
+        test_item = self.fixture_data["hits"][0]
+
+        result = self.parser.parse(test_item, provider={"config": {}})
+
+        # Parser should return a list of dicts
+        self.assertIsInstance(result, list)
+        self.assertEqual(1, len(result))
+        parsed_item = result[0]
+
+        # Check required fields
+        self.assertEqual("text", parsed_item["type"])
+        self.assertEqual("usable", parsed_item["pubstatus"])
+        self.assertIn("guid", parsed_item)
+        self.assertIn("versioncreated", parsed_item)
```

Comment:
pls test the actual values, not only that it's present, it could be wrong type or value

Reply:
can you change those 2 `assertIn` to `assertEqual` so it's visible what values are retured form the parser?